packages/lit-dev-content/site/tutorials/content/tooltip/08.md

```diff
@@ -0,0 +1,74 @@
+The tooltip supports good performance via lazy creation, but using the
+`lazy` API is cumbersome. In addition, as implemented, the tooltip content
+doesn't update when the greeted person's name changes.
+
+A Lit directive can address these issues by encapsulating the tooltip's `lazy`
```

Comment:
link out to directives docs on the words "Lit directive"

Reply:
Done